addons/addon-environment-sc-api/packages/environment-type-mgmt-services/lib/environment-type/schema/create-or-update-env-type-config.js

```diff
@@ -94,7 +95,7 @@ const schema = {
             key: {
               type: 'string', // Tag name
               maxLength: 8191,
-              pattern: '^([^<>{}]*)$',
+              pattern: nonHtmlRegex,
```

Comment:
non-block: test this validation in create

Reply:
added to CHAM-211